site/content/en/references/kustomize/cmd/localize/_index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: "kustomize localize"
+linkTitle: "localize"
+type: docs
+weight: 9
+description: >
+    Replace urls with local paths
+---
+
+Disclaimer: This is an alpha command. Please see the [command proposal](https://github.com/kubernetes-sigs/kustomize/blob/master/proposals/22-04-localize-command.md)
+for full capabilities.
+
+### Feedback
+
+Please leave your feedback for this command under [the following issue](https://github.com/kubernetes-sigs/kustomize/issues/4996).
+
+### Man
```

Comment:
Nit: I might change this to description to be more clear and closer resemble the Linux man pages

Reply:
+1. I feel like this can give users an assumption that Kustomize may have a manual pager (which is not true)

Reply:
Done. I changed `### Man` to `### Description`